src/index.js

```diff
@@ -1,9 +1,73 @@
 'use strict';
 
-const defaultOptions = {};
+const Matrix = require('ml-matrix');
+const Algebra = Matrix.algebra;
 
 function simulatedAnnealing(options) {
```

Comment:
`options` it's not a correct name for all the paramethers, you should consider to split some of them, also it's better to use an object instead of an array